LEAF_Request_Portal/admin/templates/mod_templates_email.tpl

```diff
@@ -1,643 +1,21 @@
 <link rel=stylesheet href="/libs/js/codemirror/addon/merge/merge.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.62.2/theme/lucario.min.css">
```

Comment:
This would be another spot that has the cdn file

Reply:
I didn't notice that the theme was available locally. The Library has been changed.